src/picknik_ur_mujoco_config/config/config.yaml

```diff
@@ -42,6 +42,7 @@ objectives:
       - "moveit_studio::behaviors::MTCCoreBehaviorsLoader"
       - "moveit_studio::behaviors::ServoBehaviorsLoader"
       - "moveit_studio::behaviors::VisionBehaviorsLoader"
+      - "moveit_studio::behaviors::ConverterBehaviorsLoader"
```

Comment:
I think we'll need to be more selective on what we merge into branch 6.0.
The new `ConverterBehaviorsLoader` plugin is not in the MoveIt Pro 6.0 release, so this config won't work

Reply:
Actually I think we can just drop [da43687](https://github.com/PickNikRobotics/moveit_studio_ur_ws/pull/334/commits/da436875a9a78ddcd7fe9ec8482b6d777f3de877), so that we don't require the new message builder Behaviors. This branch without that specific commit seems to work.